symphony/bdk/core/symphony_bdk.py

```diff
@@ -221,11 +222,28 @@ def presence(self) -> PresenceService:
     def activities(self) -> ActivityRegistry:
         """Get the :class:`ActivityRegistry` from the BDK entry point.
 
-        :return: The :class:`ActivityRegistry instance.
+        :return: The :class:`ActivityRegistry` instance.
 
         """
         return self._activity_registry
 
+    def slash(self, command: str, mention_bot: bool = True):
```

Comment:
Is it possible to move it inside the `activity` module?

Reply:
yes, then the example would go as following:
```python
async with SymphonyBdk(config) as bdk:
        activities = bdk.activities()
        @activities.slash("/hello")
        async def callback(context: CommandContext):
            logging.debug("Received slash command: %s", context.text_content)

        await bdk.datafeed().start()

```